### ISSUE TITLE
Loading pretrained model in pytorch

### ISSUE THREAD

**Issue by neerajkerkar** [taylover-pei/SSDG-CVPR2020#8]
How should the pretrained model be loaded? I loaded the resnet18-5c106cde.pth model provided in the pretrained_model directory into the torchvision resnet18 model. This model has an fc layer with 1000 outputs. I used the first output as the 'spoof' class score and the second output as the 'real' class score. I evaluated the model on OULU but the accuracy is very bad. Am I loading the pretrained model correctly?

**Comment by XingyuJinTI** [comment]
Please refer to https://github.com/taylover-pei/SSDG-CVPR2020/issues/9#issuecomment-680819339

**Comment by taylover-pei** [comment]
The pretrained model refers to the model pretrained on ImageNet, not the final one pretrained by the algorithm in this paper. 
Thank you.
